core/src/main/res/values/strings.xml

```diff
@@ -314,10 +314,20 @@
     <string name="enable_sonic">Enable Sonic</string>
 
     <!-- Empty list labels -->
-    <string name="no_items_label">There are no items in this list.</string>
+    <string name="no_items_header_label">No queued episodes</string>
+    <string name="no_items_label">You can add episodes to the queue by long-pressing or downloading them.</string>
     <string name="no_feeds_label">You haven\'t subscribed to any podcasts yet.</string>
     <string name="no_chapters_label">This episode has no chapters.</string>
     <string name="no_shownotes_label">This episode has no shownotes.</string>
+    <string name="no_run_downloads_head_label">No download running</string>
+    <string name="no_run_downloads_label">You can download episodes on the podcast details screen.</string>
+    <string name="no_comp_downloads_head_label">No download episodes</string>
+    <string name="no_comp_downloads_label">You can download episodes on the podcast details screen.</string>
+    <string name="no_log_downloads_head_label">No download log</string>
+    <string name="no_log_downloads_label">Download logs will appear here when available.</string>
```

Comment:
I think it's better to use `Informs about successful and failed downloads.`

Reply:
@ByteHamster What do you mean by `Informs` ?

Reply:
I wanted to tell users what exactly the download log displays instead of just telling them that the logs will appear if available. I am not a native English speaker, so maybe you have a better idea :)

Reply:
> I wanted to tell users what exactly the download log displays instead of just telling them that the logs will appear if available. I am not a native English speaker, so maybe you have a better idea :)

Do you want me to change the ```Download logs will appear here when available``` message to some meaningful message?

Reply:
Exactly. I want to make the text as meaningful as possible (while only using a single sentence).

Reply:
Alright. What about ```Download logs will be shown here with details``` or ```Download logs will be shown here with additional details like time taken, successful/unsuccessful and File URL```?

Reply:
Let's just keep the string the way it is. Sorry for changing my mind :D When adding all the information I initially wanted to add, it gets hard to read